presto-native-execution/presto_cpp/main/types/PrestoToVeloxExpr.cpp

```diff
@@ -102,6 +80,15 @@ std::string getFunctionName(const protocol::SqlFunctionId& functionId) {
 
 } // namespace
 
+const std::unordered_map<std::string, std::string>
+veloxToPrestoScalarOperatorMap() {
+  std::unordered_map<std::string, std::string> scalarOperatorMap;
```

Comment:
The keys and values in the map are reversed here since we need to map velox scalar operator name to that in Presto. So this is returning an inverse map of `kFunctionNames` (renamed now to `kPrestoOperatorMap`), which is also moved to the header. Could you please share if this is fine?